src/bindings/python/src/pyopenvino/core/properties/properties.cpp

```diff
@@ -101,6 +101,8 @@ void regmodule_properties(py::module m) {
     wrap_property_RW(m_hint, ov::hint::allow_auto_batching, "allow_auto_batching");
     wrap_property_RW(m_hint, ov::hint::dynamic_quantization_group_size, "dynamic_quantization_group_size");
     wrap_property_RW(m_hint, ov::hint::kv_cache_precision, "kv_cache_precision");
+    wrap_property_RW(m_hint, ov::hint::key_cache_group_size, "key_cache_group_size");
```

Comment:
please add them also [here](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/runtime/properties/hint/__init__.py#L25)
and add a test [here](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_runtime/test_properties.py)